applications/asset_tracker_v2/thingy91_nrf9160ns.overlay

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/ {
+        aliases {
+                temp-sensor = &bme680;
+                humidity-sensor = &bme680;
+                accelerometer = &adxl362;
+        };
+};
+
+&i2c2 {
+	bme680: bme680@76 {};
```

Comment:
```
	bme680: bme680@76 {
		compatible = "bosch,bme680";
		reg = <0x76>;
		label = "BME680";
	};
```

Reply:
I sort of hoped that there was some magic in the background that mapped this to the properties defined in https://github.com/nrfconnect/sdk-nrf/blob/c468349c45e1862d51d94cca3eeb41a782965934/boards/arm/thingy91_nrf9160/thingy91_nrf9160_common.dts#L114
And it actually works without it, so that's curious, or maybe I did something funky when testing it. Anyway, I'll add it and update the PR.

Reply:
Added

Reply:
Oh, I see! My mistake. Yes, your original PR was OK then. Sorry for the noise. Feel free to go back.

Reply:
No worries, I'll just revert it back to not have to maintain copies of those configurations even if they'll probably not change anytime soon